bitnami/minio/Chart.yaml

```
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/minio
   - https://min.io
-version: 12.4.3
+version: 12.4.4
```

As you said is a breaking change so I think we should update a mayor version
```suggestion
version: 13.0.0
```